packages/mantine/src/components/table/layouts/__tests__/RowLayout.spec.tsx

```diff
@@ -398,13 +399,14 @@ describe.skip('RowLayout', () => {
                     disableRowSelection
                 />,
             );
-
-            expect(screen.getByRole('checkbox', {name: /select all/i})).toHaveStyle('pointerEvents: none');
+            screen.getByRole('checkbox', {name: /select all/i}).click();
```

Comment:
Shouldn't we be using `user.click(element)` instead of `element.click()` ?

Reply:
Woops that's my bad